x/mongo/driver/session/session_pool.go

```diff
@@ -20,13 +20,20 @@ type Node struct {
 	prev *Node
 }
 
+// topologyDescription is used to track a subset of the fields present in a description.Topology instance that are
```

Comment:
```suggestion
// topologyDescription is used to track a subset of the fields present in a description. Topology instance that are
```

Reply:
This is correct as-is because `description.Topology` is the name of a type so the `.` is not meant to be a period here.